index.html

```diff
@@ -44,9 +44,7 @@
         </p>
       </div>
 
-      <div class="connect">
-        <p>LET’S CONNECT</p>
-      </div>
+      <p class="connect">LET’S CONNECT</p>
 
       <div class="social-icons">
         <span class="iconify" data-icon="iconoir:github"></span>
```

Comment:
- [ ] The main title ("I'm Ismail...)is should be the `h1` tag. In this case, you are using the `p` tag. Please try to fix this issue.

<img width="311" alt="Screen Shot 2021-11-24 at 12 50 47 PM" src="https://user-images.githubusercontent.com/69011963/143201370-e68df0de-bf86-4c4f-8f97-21cf49f74aaa.png">